NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(aval_resp)
+export(call_aval_fun)
 export(death_event)
```

Comment:
I can see this helps us as developers, but should we export this? Or is it convention to expose all?

Reply:
I would export it because it could be useful for implementing company specific functions.